mitiq/pec/representations/biased_noise.py

```diff
@@ -0,0 +1,148 @@
+# Copyright (C) 2022 Unitary Fund
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+"""Function to generate representations with biased noise."""
+
+from typing import List
+
+from cirq import (
+    Operation,
+    X,
+    Y,
+    Z,
+    Circuit,
+)
+
+from mitiq import QPROGRAM
+from mitiq.pec import OperationRepresentation, NoisyOperation
+from mitiq.interface import convert_to_mitiq, convert_from_mitiq
+
+
+def represent_operation_with_local_biased_noise(
+    ideal_operation: QPROGRAM,
+    epsilon: float,
+    eta: float,
+) -> OperationRepresentation:
+    r"""This function maps an
+    ``ideal_operation`` :math:`\mathcal{U}` into its quasi-probability
+    representation, which is a linear combination of noisy implementable
+    operations :math:`\sum_\alpha \eta_{\alpha} \mathcal{O}_{\alpha}`.
+
+    This function assumes a combined depolarizing and dephasing noise model
+    with a bias factor :math:`\eta` (see :cite:`Strikis_2021_PRXQuantum`)
+    and that the following noisy operations are implementable
+    :math:`\mathcal{O}_{\alpha} = \mathcal{D} \circ \mathcal P_\alpha`
+    where :math:`\mathcal{U}` is the unitary associated
+    to the input ``ideal_operation``,
+    :math:`\mathcal{P}_\alpha` is a Pauli operation and
+    ..math::`\mathcal{D}(\epsilon) = (1 - \epsilon)[mathbb{1}] +
+    \epsilon(frac{\eta}{\eta + 1} mathcal{Z}
+    + frac{1}{3}frac{1}{\eta + 1}(mathcal{X} + mathcal{Y} + mathcal{Z}))` is
```

Comment:
This is not rendering well on readthedocs (https://mitiq--1233.org.readthedocs.build/en/1233/apidoc.html#module-mitiq.pec.representations.optimal). Maybe an empty line before and after the equation is necessary and/or maybe a space between `.. `and `math::` ?

Reply:
Moreover I think `mathbb`, `mathcal`, `frac` require a preceding `\` char.